content/MLaaS4HEP.md

```diff
@@ -0,0 +1,94 @@
+## Machine Learning as a Service for HEP
+
+MLaaS for HEP is a set of Python based modules to support reading HEP data and
+stream them to ML of user choice for training. It consists of three independent layers:
+- Data Streaming layer to handle remote data, see [reader.py](https://github.com/vkuznet/MLaaS4HEP/blob/master/src/python/MLaaS4HEP/reader.py)
+- Data Training layer to train ML model for given HEP data, see [workflow.py](https://github.com/vkuznet/MLaaS4HEP/blob/master/src/python/MLaaS4HEP/workflow.py)
+- Data Inference layer, see [tfaas_client.py](https://github.com/vkuznet/TFaaS/blob/master/src/python/tfaas_client.py)
+
+The MLaaS4HEP resopitory can be found [here](https://github.com/vkuznet/MLaaS4HEP).
+
+The general architecture of MLaaS4HEP looks like this:
+![MLaaS4HEP-architecture](https://github.com/vkuznet/MLaaS4HEP/blob/master/images/MLaaS4HEP_arch_gen.png)
+
+Even though this architecture was originally developed for dealing with
+HEP ROOT files we extend it to other data formats. So far the following
+data formats are supported: JSON, CSV, Parquet, ROOT. The former ones support
+reading files from local file system or HDFS, while later (ROOT) format allows
+to read ROOT files from local file system or remote files via xrootd protocol.
+
+The pre-trained models can be easily uploaded to
+[TFaaS](https://github.com/vkuznet/TFaaS) inference server for serving them to clients.

```

Comment:
Can you add a line about TFaaS being documented at `content/inference/tfaas.md`. As soon as #13 is merged that file will be available. That way these two documents are cross-referenced.